### PR TITLE
use cross-spawn

### LINKED ISSUE #3: use node-cross-spawn
Can we use [node-cross-spawn](https://github.com/IndigoUnited/node-cross-spawn) for windows folks?

I've managed to install fontcustom on windows (not easy, though). The only thing missing is for node to correctly spawn the process without any errors.

### PR DESCRIPTION
**Pull request by miguelcobain**
closes #3

### CODE REVIEW

**Comment by miguelcobain** [comment]
@Kinvey Can you merge this and release on NPM?

This would make a huge difference for Windows folks.

**Comment by miguelcobain** [comment]
Ping.

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "broccoli-kitchen-sink-helpers": "^0.2.5",
     "coffee-script": "^1.7.1",
+    "cross-spawn": "^2.0.0",
```
Comment (patrickjholloway) [inline package.json:28]: Perhaps this should be bumped to a more recent version, cross-spawn has been updated to 6.0.5

**Review by patrickjholloway — CHANGES_REQUESTED** [review]
@miguelcobain curious if you have the time to include an update to the README for instructions to install on Windows, if not I can take a stab at it when I have a chance.

**Comment by miguelcobain** [comment]
@patrickjholloway unfortunately I'm no longer on a windows machine, but I remember this being a relatively safe change to make.

File: package.json (hunk 1 above)
Comment (miguelcobain) [reply package.json:28]: I think it should be fine, yes. Looks like the api didn't change at all.

**Comment by patrickjholloway** [comment]
@miguelcobain okay, sounds good. I do want to test this out and add an entry to the README. I will take care of this soon and merge. Thank you for the PR and sorry for the huge delay.

**Comment by dimofeev** [comment]
@patrickjholloway do we have any progress here?

**Comment by dimofeev** [comment]
@patrickjholloway any news here ?

**Comment by dimofeev** [comment]
Going to close the PR. Will be reopened in case it is needed at some point.

**Pull request closed without merging**